packages/vuetify/src/labs/VSkeletonLoader/VSkeletonLoader.tsx

```diff
@@ -0,0 +1,147 @@
+// Styles
+import './VSkeletonLoader.sass'
+
+// Composables
+import { makeDimensionProps, useDimension } from '@/composables/dimensions'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeThemeProps, provideTheme } from '@/composables/theme'
+import { useBackgroundColor } from '@/composables/color'
+
+// Utilities
+import { computed, h, toRef } from 'vue'
+import { genericComponent, useRender } from '@/util'
+
+// Types
+import type { PropType, VNode } from 'vue'
+
+export const rootTypes = {
+  actions: 'button@2',
+  article: 'heading, paragraph',
+  avatar: 'avatar',
+  button: 'button',
+  card: 'image, heading',
+  'card-avatar': 'image, list-item-avatar',
+  chip: 'chip',
+  'date-picker': 'list-item, heading, divider, date-picker-options, date-picker-days, actions',
+  'date-picker-options': 'text, avatar@2',
+  'date-picker-days': 'avatar@28',
+  heading: 'heading',
+  image: 'image',
+  'list-item': 'text',
+  'list-item-avatar': 'avatar, text',
+  'list-item-two-line': 'sentences',
+  'list-item-avatar-two-line': 'avatar, sentences',
+  'list-item-three-line': 'paragraph',
+  'list-item-avatar-three-line': 'avatar, paragraph',
+  paragraph: 'text@3',
+  sentences: 'text@2',
+  subtitle: 'text',
+  table: 'table-heading, table-thead, table-tbody, table-tfoot',
+  'table-heading': 'chip, text',
+  'table-thead': 'heading@6',
+  'table-tbody': 'table-row-divider@6',
+  'table-row-divider': 'table-row, divider',
+  'table-row': 'text@6',
+  'table-tfoot': 'text@2, avatar@2',
+  text: 'text',
+} as Record<string, string>
+
+function genBone (text: string, children: VNode[] = []) {
+  return h('div', {
```

Comment:
any special reason for not using tsx?

Reply:
Wasn't sure how to render the array of vnodes using it, open to suggestions

Reply:
Maybe I'm tired and not thinking straight, but shouldn't it just be
```js
<div class={[...]}>{ children }</div>
```